examples/plugin/reset_plugin/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+
+find_package(ignition-cmake2 REQUIRED)
+
+project(ResetPlugins)
+find_package(ignition-gazebo7 REQUIRED)
+add_library(JointPositionRandomizer SHARED joint_position_randomizer.cc)
```

Follow the pattern of `CamelCase` for plugin files

```suggestion
add_library(JointPositionRandomizer SHARED JointPositionRandomizer.cc)
```